examples/hybrid/tls_shared.tf

```diff
@@ -5,7 +5,6 @@ resource "tls_private_key" "ca" {
 }
 
 resource "tls_self_signed_cert" "ca" {
-  key_algorithm     = tls_private_key.ca.algorithm
```

Comment:
`tls_private_key.ca.algorithm` has been deprecated - theres no `required_providers` block in any of the examples, so it uses the latest aws provider `4.23`. Instead of a warning, terraform now produces an error when attempting to provision

Do we not still need the key to create the self signed cert/key pair?